src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Presentation/CompletionPresenter.cs

```diff
@@ -45,7 +45,7 @@ internal sealed class CompletionPresenter : ForegroundThreadAffinitizedObject, I
 
             var document = subjectBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
 
-            var completionSetFactory = document != null && NeedsDev15CompletionSetFactory(document.Options, document.Project.Language)
+            var completionSetFactory = document != null && NeedsDev15CompletionSetFactory(document.Project.Solution.Options, document.Project.Language)
```

Comment:
how can Solution directly expose options?  wouldn't is be affected by .editorconfig files above the .sln file?